features-json/css-grid-animation.json

```diff
@@ -0,0 +1,491 @@
+{
+	"title":"CSS Grid animation",
+	"description":"As defined in CSS Grid level 1, grid-template-columns and grid-template-rows are properties that can be animated.",
+	"spec":"https://www.w3.org/TR/css-grid-1/",
+	"status":"cr",
+	"links":[
+		{
```

Comment:
I think it's the empty "link" that it's got a problem with, just need to change this to an empty array. :)

Reply:
Thanks. Fixed. Let's see if that passes.